features/git-sync/all_branches/feature_branch_merge_main_branch_conflict/without_tracking_branch_updates/last_branch/without_remote_origin.feature

```diff
@@ -0,0 +1,106 @@
+Feature: git sync --all: handling merge conflicts between feature branch and main branch without open changes or remote origin
+
+  Background:
+    Given I have feature branches named "feature1" and "feature2"
+    And my repo does not have a remote origin
+    And the following commits exist in my repository
+      | BRANCH   | LOCATION | MESSAGE         | FILE NAME        | FILE CONTENT     |
+      | main     | local    | main commit     | conflicting_file | main content     |
+      | feature1 | local    | feature1 commit | feature1_file    | feature1 content |
+      | feature2 | local    | feature2 commit | conflicting_file | feature2 content |
+    And I am on the "main" branch
+    When I run `git sync --all`
+
+
+  Scenario: result
+    Then it runs the Git commands
+      | BRANCH   | COMMAND                  |
+      | main     | git checkout feature1    |
+      | feature1 | git merge --no-edit main |
+      | feature1 | git checkout feature2    |
+      | feature2 | git merge --no-edit main |
+    And I get the error
+      """
+      To abort, run "git sync --abort".
+      To continue after you have resolved the conflicts, run "git sync --continue".
+      To skip the sync of the 'feature2' branch, run "git sync --skip".
+      """
+    And I end up on the "feature2" branch
+    And my repo has a merge in progress
+
+
+  Scenario: aborting
+    When I run `git sync --abort`
+    Then it runs the Git commands
+      | BRANCH   | COMMAND                                |
+      | feature2 | git merge --abort                      |
+      | feature2 | git checkout feature1                  |
+      | feature1 | git reset --hard [SHA:feature1 commit] |
```

Comment:
I think this is the correct behavior for abort, but it differs from the remote case because feature1 will have already been pushed and won't be reset back.